org.opensurge2d.OpenSurge.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: org.opensurge2d.OpenSurge
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
+sdk: org.freedesktop.Sdk
+command: opensurge
+rename-icon: opensurge
+rename-desktop-file: opensurge.desktop
```

Comment:
These should probably renamed upstream otherwise software centers could get confused and show up the game is available from two different sources. See the specs in https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Reply:
These files are named like some other games that are also on flathub, such as SuperTux and SuperTuxKart. The app-id is still unique. Thank you for pointing out.